src/io/jesi/backpack/test/runner.cljs

```diff
@@ -19,8 +21,11 @@
   (st/run-all-tests (ctd/init! "test-root")))
 
 (defn ^:dev/before-load-async stop [done]
-  ;FIXME determine if async tests are still pending https://github.com/clojure/clojurescript-site/blob/master/content/tools/testing.adoc#detecting-test-completion--success
-  (done))
+  (async/go
+    (println "Waiting for tests to complete...")
+    ;TODO cancel async tests
```

Comment:
Not sure how to do this. I don't think cljs.test stores a reference to the async test result